cura/Machines/MaterialManager.py

```diff
@@ -430,27 +170,10 @@ def getMaterialNode(self, machine_definition_id: str, nozzle_name: Optional[str]
     #
     def getMaterialNodeByType(self, global_stack: "GlobalStack", position: str, nozzle_name: str,
```

Comment:
`nozzle_name` is not used here. `variant_name` is probably the same as `nozzle_name`.

Reply:
Yes, but this API shouldn't change. It's deprecated though.